etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamOrderSenderTest.groovy

```diff
@@ -26,6 +27,7 @@ class ReportStreamOrderSenderTest extends Specification {
         TestApplicationContext.reset()
         TestApplicationContext.init()
         TestApplicationContext.register(OrderSender, ReportStreamOrderSender.getInstance())
+        TestApplicationContext.register(MetricMetaData, Mock(MetricMetaData))
```

Comment:
In this test file here and above, we just register the `MetricMetaData` in the application context.  I feel we should make some tests that assert that we call the `MetricMetaData#put` method.  For example, if I accidentally removed the `metaData.put(order.getFhirResourceId(), EtorMetaDataStep.SENT_TO_REPORT_STREAM);` line from `ReportStreamOrderSender`, I would want a test to fail.  This applies to the other unit tests were we call our metadata.

Reply:
done.